### PR TITLE
Limit concurrent API calls

### LINKED ISSUE #580: Be wary of rate limits
https://github.com/withspectrum/spectrum/issues/2632

Looks like when a PR gets big enough, danger and peril can hit the rate limits. Will need to slow down diff/comment pagination probably

### PR DESCRIPTION
**Pull request by mxstbr**
Closes #580

### CODE REVIEW

File: source/platforms/github/GitHubAPI.ts [hunk 1]
```diff
@@ -15,6 +16,8 @@ import { RepoMetaData } from "../../dsl/BitBucketServerDSL"
 
 export type APIToken = string
 
+const limit = pLimit(25)
```
Comment (mxstbr) [inline source/platforms/github/GitHubAPI.ts]: Note: open to different limits here, this is just a first shot in the dark. Not sure if this'll even fix the issue. 😅

**Comment by orta** [comment]
Dang, I wonder how you can get a null pr body, `danger.github.pr.body.includes("peril-debug")`

Regardlesss. Yeah, this looks good 👍

**Comment by orta** [comment]
Merge on green - and I'll make a release

**Comment by mxstbr** [comment]
> Dang, I wonder how you can get a null pr body

Submitted PR from the command line with no body 😅

**Comment by mxstbr** [comment]
Wait does "Merge on green" only add the label automatically or also automatically merge it once CI passes? 😱

**Comment by orta** [comment]
It does both! - https://twitter.com/orta/status/990214980871098371

File: source/platforms/github/GitHubAPI.ts (hunk 1 above)
Comment (orta) [reply source/platforms/github/GitHubAPI.ts]: It's a good enough start for me

**Comment by mxstbr** [comment]
Damnnnnn 🔥🔥🔥🔥🔥

**Pull request merged**